expressEngine/lib/parser/base.js

```diff
@@ -5,12 +5,36 @@ class Parser {
       err.message ||
       err;
     return {
-      message: errMsg,
-      error: true,
+      error: errMsg,
+    };
+  }
+
+  success = (data, successMsg = false) => {
```

Comment:
No use of default parameter `(successMsg = false)`
Because on line no. 20 we are adding the condition `message: successMsg || 'Process compeleted successfully',`
Instead, we can do this `success = (data, successMsg = "Process compeleted successfully") => {`